### PR TITLE
Rustup

### PR DESCRIPTION
**Pull request by lzutao**
changelog: none

### CODE REVIEW

File: clippy_lints/src/methods/mod.rs [hunk 1]
```diff
@@ -1773,7 +1773,7 @@ fn derefs_to_slice<'a, 'tcx>(
     expr: &'tcx hir::Expr,
     ty: Ty<'tcx>,
 ) -> Option<&'tcx hir::Expr> {
-    fn may_slice(cx: &LateContext<'_, '_>, ty: Ty<'_>) -> bool {
+    fn may_slice<'a>(cx: &LateContext<'_, 'a>, ty: Ty<'a>) -> bool {
```
Comment (phansch) [inline clippy_lints/src/methods/mod.rs]: I'm not sure, but shouldn't the `Ty` lifetime be called `'tcx`?

Comment (lzutao) [reply clippy_lints/src/methods/mod.rs]: Yes, but does it conflict with outer function?

Comment (phansch) [reply clippy_lints/src/methods/mod.rs]: Oh, I didn't see that it was an inner function. I think then it's fine =)

**Pull request merged**